src/component/Footer.svelte

```diff
@@ -0,0 +1,153 @@
+<script>  
+    let srcMilk = '././img/image-gallery-milkbottles.jpg'; 
+    let srcOrange = '././img/image-gallery-orange.jpg';
+    let srcCone = '././img/image-gallery-cone.jpg';
+    let srcSugar = '././img/image-gallery-sugarcubes.jpg'; 
+
+    let srcLogo = '././img/sunnyside.svg'; 
+
+    let srcFacebook = '././img/icon-facebook.svg';
+    let srcInstagram = '././img/icon-instagram.svg';
+    let srcTwitter = '././img/icon-twitter.svg';
+    let srcPinterest = '././img/icon-pinterest.svg'; 
+</script>
+
+<footer>
+    <div class='footer-img'>
+        <img src={srcMilk} class='srcMilk' alt='MilkPhoto'>
```

Comment:
asagida style kismindan css'le style eklemeyeceksen `class` vermene gerek yok. Kullanmadigin `class`lari silelim.

Reply:
47.satırda bu class'a style eklemiştim.
```suggestion
        <img src={srcMilk} class='srcMilk' alt='MilkPhoto'>
```